tests/protocol/test_wire.py

```diff
@@ -59,6 +59,22 @@ def _test_getters(self, test_data, MockCryptUtil, _):
 
             self.assertEqual("1", protocol.get_incarnation())
 
+    def test_serialize_users(self, mock_cryptutil, mock_sleep):
+        users = None
+        expected = ''
+        actual = serialize_users(users)
+        self.assertEqual(expected, actual)
+
+        users = []
+        actual = serialize_users(users)
+        self.assertEqual(expected, actual)
+
+        user1 = ["testUser1", None, "2030-12-12, None"]
+        users.append(user1)
+        expected = '<User>testUser1</User>'
```

Comment:
You throw out quite a bit of information.  Is this intentional?

Reply:
removed